Cargo.toml

```
@@ -7,13 +10,17 @@ members = [
     "crates/brioche-packed-plain-exec",
     "crates/brioche-packed-userland-exec",
     "crates/brioche-packer",
-    "crates/brioche-resources", "crates/brioche-strip",
+    "crates/brioche-resources",
+    "crates/brioche-strip",
     "crates/runnable-core",
 ]
 
 [workspace.dependencies]
 brioche-pack = { git = "https://github.com/brioche-dev/brioche.git", default-features = false }
 
+[workspace.lints.clippy]
+all = { level = "deny", priority = -1 }
+
```

Same thoughts here as on the Brioche PR: https://github.com/brioche-dev/brioche/pull/164#discussion_r1921684669

```suggestion
[workspace.lints.clippy]
all = { level = "warn", priority = -1 }

```

---

I updated it.